src/app/util/attribute-storage.cpp

```diff
@@ -139,7 +138,7 @@ void emberAfEndpointConfigure()
 
     emberEndpointCount                = FIXED_ENDPOINT_COUNT;
     DataVersion * currentDataVersions = fixedEndpointDataVersions;
-    for (ep = 0; ep < FIXED_ENDPOINT_COUNT; ep++)
+    for (ep = 0; ep < emberEndpointCount; ep++)
```

Comment:
Why this change?  Given line 139, this should be no different, I would think, unless something inside the loop changes `emberEndpointCount`... but if it does, that's because someone is adding dynamic endpoints, and we do _not_ want to do this init for dynamic endpoints: it will cause out-of-bounds array accesses!